cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -159,6 +159,9 @@
     {% include 'templates/header.html' %}
 {% endblock header %}
 
+<!-- WAGTAIL USERBAR IN PAGE CONTROLS -->
```

Comment:
Consider converting to jinja comment as `{# WAGTAIL USERBAR IN PAGE CONTROLS #}` unless there's a reason to show this in the output HTML.